infra/net/handlers.go

```diff
@@ -0,0 +1,37 @@
+package net
+
+import (
+	"bootcamp/adapters"
+	"bootcamp/helper"
+	"fmt"
+	"net/http"
+)
+
+func NewGetCharacterHandler(getCharacterController adapters.GetCharacterController) func(writer http.ResponseWriter, request *http.Request) {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		id, err := helper.ExtractId("/character/", request.URL.Path)
```

Comment:
The [mux router](https://github.com/gorilla/mux) can help you to avoid doing operations like this manually.

Reply:
Thx, I will use it